pkg/composableschemadsl/compiler/importer.go

```diff
@@ -0,0 +1,54 @@
+package compiler
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/authzed/spicedb/pkg/composableschemadsl/input"
+	"github.com/authzed/spicedb/pkg/genutil/mapz"
+)
+
+type ImportContext struct {
+	pathSegments []string
+	sourceFolder string
+	names        *mapz.Set[string]
+}
+
+const SchemaFileSuffix = ".zed"
+
+func ImportFile(importContext *ImportContext) (*CompiledSchema, error) {
```

Comment:
Unless you see a need to export this, should we just keep this internal (i.e. lowercase)?

Reply:
That works for me